docs/Development/Start new OpenY project.md

```diff
@@ -1,25 +1,275 @@
-Start new OpenY project
-=====
-
 Here you can find instructions how you can start project based on OpenY distribution.
 
+# New project from scratch based on OpenY
+
+In order to start new project from scratch, you can use [installation instructions](https://github.com/ymcatwincities/openy-project#installation) that will build your project and even add development environment.
+
+
+# Add OpenY to existing Drupal 8 project
+
+Please take a look at the full `composer.json` file below that you should eventually get.
+<details> 
+  <summary><strong>Example composer.json (Drupal 8.3.2)</strong></summary>
+
+```
+{
+    "name": "drupal/drupal",
+    "description": "Drupal is an open source content management platform powering millions of websites and applications.",
+    "type": "project",
+    "license": "GPL-2.0+",
+    "require": {
+        "composer/installers": "^1.0.24",
+        "wikimedia/composer-merge-plugin": "~1.4",
+        "ymcatwincities/openy": "8.*.*",
+        "cweagans/composer-patches": "~1.0"
+    },
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "config": {
```

Comment:
I was able to reproduce this on ubuntu.
![image](https://user-images.githubusercontent.com/1316234/27857084-04041b46-6179-11e7-9a86-5f71cc0cec37.png)